doc/source/getting_started/XarrayforArviZ.ipynb

```diff
@@ -12,17 +12,17 @@
    "cell_type": "markdown",
```

Comment:
<p>I would aim for link plus 1-2 sentences explaining what they'll find there. i.e. I am not completely sure what the basics of netCDF are, but the link directs me to the netcdf4-python library, a low level library to read/write netcdf files. The 2nd link however goes to xarray io page about read/write netcdf files directly as xarray objects. </p><p> </p><p>side note, the link to xarray page can be </p><p> </p><pre class="ql-syntax" spellcheck="false">{ref}<code>xarray:io.netcdf</code>
</pre><p> </p><p>see first sentence in https://github.com/pydata/xarray/blob/main/doc/user-guide/io.rst or use <code>sphobjinv</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/arviz-devs/arviz/pull/1782/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>